scraper/gen_code_gov_json.py

```diff
@@ -99,6 +137,26 @@ def process_doecode(doecode_json_filename):
     return projects
 
 
+def government_at_github():
+    """
+    Returns a list of US Government GitHub orgs
+
+    Based on: https://government.github.com/community/
+    """
+    us_gov_github_orgs = set()
+
+    gov_yml = requests.get('https://raw.githubusercontent.com/github/government.github.com/gh-pages/_data/governments.yml')
```

Comment:
There's also https://government.github.com/organizations.json if it's helpful.

Reply:
ooo... Didn't know that was a thing! That looks better than my approach. Didn't realize that the data was exposed there!

I'll go ahead and work that into a future pull request.. Thanks for the heads up!

Reply:
for reference: https://jekyllrb.com/docs/datafiles/

Reply:
Updated! https://github.com/LLNL/scraper/commit/585dc63dba5232ef93801d322a2c0118f618756d